developerportal/apps/events/models.py

```diff
@@ -375,6 +375,51 @@ class Event(BasePage):
         help_text="Optional list of speakers for this event",
     )  # DEPRECATED
 
+    extra_content_panel_title = CharField(
```

Comment:
This files' changes are from another PR which was based off this one, which I should have held off merging, but didn't... Sorry.

Ignore for review